bundle/Resources/config/role.yml

```diff
@@ -177,3 +180,18 @@ services:
         arguments: [%ezrepoforms.limitation.form_mapper.parentdepth.maxdepth%]
         tags:
             - { name: ez.limitation.formMapper, limitationType: ParentDepth }
+
+    ezrepoforms.limitation.form_mapper.location:
+        class: %ezrepoforms.limitation.form_mapper.location.class%
+        tags:
+            - { name: ez.limitation.formMapper, limitationType: Node }
+        calls:
+            - [setFormTemplate, [%ezrepoforms.limitation.udw.template%]]
+
+    ezrepoforms.limitation.form_mapper.subtree:
+        class: %ezrepoforms.limitation.form_mapper.subtree.class%
+        arguments: [@ezpublish.api.service.location]
+        tags:
+            - { name: ez.limitation.formMapper, limitationType: Subtree }
+        calls:
+            - [setFormTemplate, [%ezrepoforms.limitation.udw.template%]]
```

Comment:
@lolautruche says: As already suggested (IIRC), this call and the same one for the location mapper can be abstracted in an abstract service.